src/pages/help-centre/_utility.js

```diff
@@ -1,5 +1,16 @@
-export const convertToHash = (category, label) => {
-    return '/help-centre/' + category.replace(/\s/g, '-').toLowerCase() + '/#' + label
+export const convertToHash = (category, label, qparam) => {
+    if (qparam) {
+        return (
+            '/help-centre/' +
+            category.replace(/\s/g, '-').toLowerCase() +
+            '/?platform=' +
+            qparam +
+            '#' +
+            label
```

Comment:
Can we have this a template literal and abstract out the replace function?

Reply:
Sure, That would be better. Let me try.